unittests/files/opt/seagate/sspl/setup/resource_map/test_storage.py

```diff
@@ -130,6 +130,57 @@
     """
 ENCLOSURE_RESPONSE_EMPTY = {}
 
+ENCLOSURE_NW_RESPONSE = """
```

Comment:
Please rebase and move this response to encl_response.py file.

Reply:
Addressed 👍